kiwi/xml_state.py

```diff
@@ -923,7 +966,7 @@ def get_volumes(self):
 
         * name: name of the volume
         * size: size of the volume
-        * realpath: system path to lookup volume data. If no mountpoint
+        * realpath: system h to lookup volume data. If no mountpoint
```

Comment:
sounds like a mistake ?

Reply:
Yes  mistake... shame on me. Should stop doing any PR just right before lunch.

Reply:
nevermind that just tests if I take the review serious ;)